addon/decorators/controller-pagination.ts

```diff
@@ -1,13 +1,12 @@
 import DS from 'ember-data';
```

Comment:
these `addon/decorators/...` files should be removed completely, in favor of only having the `addon/mixins/..` typescript mixin versions.